### PR TITLE
Refactor the route definition file 

### PR DESCRIPTION
**Pull request by MauricioFauth**
- Move request and response handling from routes file to the controllers.
- Simplify the routes file by moving the controller creation to the `index.php` file.
- Other minor changes.

### CODE REVIEW

File: libraries/classes/Controllers/Database/StructureController.php [hunk 1]
```diff
@@ -219,22 +216,26 @@ public function addRemoveFavoriteTablesAction(array $parameters): ?array
...
-            return [];
+            return;
         }
         $changes = true;
         $titles = Util::buildActionTitles();
         $favoriteTable = $parameters['favorite_table'] ?? '';
         $alreadyFavorite = $this->checkFavoriteTable($favoriteTable);
 
-        if (isset($parameters['remove_favorite'])) {
+        if (isset($_REQUEST['remove_favorite'])) {
```
Comment (williamdes) [inline libraries/classes/Controllers/Database/StructureController.php]: _POST ?

**Review by williamdes — APPROVED** [review]
:100:

File: libraries/classes/Controllers/Database/StructureController.php (hunk 1 above)
Comment (mauriciofauth) [reply libraries/classes/Controllers/Database/StructureController.php]: It was `$_REQUEST['remove_favorite']` in the `routes.php`, but I agree it should be POST. However I think it should be done in other PR. And I think it's better to create a new route for this as well.

Comment (williamdes) [reply libraries/classes/Controllers/Database/StructureController.php]: Okay, I agree :)
Nice PR !

**Pull request merged**